### PR TITLE
Added support for gallery view

### PR DESCRIPTION
**Pull request by itstillon**
## What changed
- Added support to send image of the latest challenges taken by users.
- Modified `Story` to fetch `StoryImage`s eagerly since it would be required for gallery view as well as admin workflow for reviewing submitted stories.

### CODE REVIEW

**Comment by itstillon** [comment]
FYI, it is not verified with UI. The API provided is based on the pattern followed for Spring MVC.

File: ideaforce/src/main/java/com/emc/ideaforce/controller/CommonController.java [hunk 1]
```diff
@@ -129,9 +133,25 @@ public ModelAndView submitStory(@RequestParam String challengeId,
         return mv;
     }
 
+    @ResponseBody
     @GetMapping("/gallery")
-    public String gallery() {
-        return GALLERY_VIEW;
+    public List<byte[]> gallery() {
```
Comment (itstillon) [inline ideaforce/src/main/java/com/emc/ideaforce/controller/CommonController.java]: Vaibhav, can you confirm if this API signature would work with Thymeleaf?

**Comment by quailbr** [comment]
@itstillon ,
Shall I merge once tested?

**Comment by itstillon** [comment]
Yes, better to wait for Vaibhav to confirm.

**Comment by itstillon** [comment]
Opened #13 PR containing these changes and profile page related changes. Hence closing this PR.

**Pull request closed without merging**